pkg/util/phone/parser_test.go

```diff
@@ -235,3 +238,27 @@ func TestParse_IsPossibleNumber_ReturnE164(t *testing.T) {
 		test(" +852 9876 5432 ", "")
 	})
 }
+
+func TestCountryCallingCodeToRegions(t *testing.T) {
+	Convey("Country calling codes to regions", t, func() {
+		c := phonenumbers.GetSupportedCallingCodes()
+		m := map[int][]string{}
+		for countryCallingCode := range c {
+			codes := phonenumbers.GetRegionCodesForCountryCode(countryCallingCode)
+			m[countryCallingCode] = codes
+		}
+		data, err := os.ReadFile("calling_codes_regions.json")
+		if err != nil {
+			panic(err)
+		}
+
+		var expectedMap map[int][]string
+
+		err = json.Unmarshal(data, &expectedMap)
+		if err != nil {
+			panic(err)
+		}
+
+		So(m, ShouldEqual, expectedMap)
```

Comment:
Shouldn't we use `ShouldResemble` here? AFAIK, ShouldEqual uses `==` which cannot be used for comparing maps.

Reply:
https://github.com/smarty/assertions/commit/e64f0e2935168e4829711112dc859c21e2e85e27